pkgs/development/tools/rust/cargo-llvm-cov/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, rustPlatform, fetchCrate, fetchFromGitHub, libllvm, makeBinaryWrapper }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-llvm-cov";
+  version = "0.4.14";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-DY5eBSx/PSmKaG7I6scDEbyZQ5hknA/pfl0KjTNqZlo=";
+  };
+
+  nativeBuildInputs = [ makeBinaryWrapper ];
+  buildInputs = [ libllvm ];
+
+  doCheck = false;
```

The tests can still be ran, but only a subset of them:

```suggestion
  src = fetchCrate {
    inherit pname version;
    sha256 = "sha256-DY5eBSx/PSmKaG7I6scDEbyZQ5hknA/pfl0KjTNqZlo=";
  };

  postPatch = ''
    rm test/tests.rs
  '';

  nativeBuildInputs = [ makeBinaryWrapper ];
  buildInputs = [ libllvm ];
```